src/test/java/com/wikia/webdriver/common/contentpatterns/MercurySubpages.java

```diff
@@ -29,6 +29,11 @@ private MercurySubpages() {}
 
   // Articles prepared for mercurycc.wikia.com wiki (with FC, CC, TA and TV)
   public static final String CC_MAIN_PAGE = "Mercury_CC_Wikia";
+  // Categories in mercurycc.wikia.com
+  public static final String CC_CATEGORY_ARTICLES = "main/category/Articles";
+  public static final String CC_CATEGORY_BLOGS = "main/category/Blog_posts";
+  public static final String CC_CATEGORY_IMAGES = "main/category/Images";
+  public static final String CC_CATEGORY_VIDEOS = "main/category/Videos";
```

Comment:
how about creating Class for all constants related to curated main pages? so it is nicely grouped.